ambry-network/src/main/java/com/github/ambry/network/Selector.java

```diff
@@ -243,6 +244,9 @@ public void close() {
       metrics.selectorNioCloseErrorCount.inc();
       logger.error("Exception closing nioSelector:", e);
     }
+    if (executorPool != null) {
+      executorPool.shutdown();
```

Comment:
Good fix. Btw, could you format this file? (At least `import java.util.concurrent.Executors;` can be removed)